library/prelude/path.red

```diff
@@ -105,6 +119,18 @@ def symmd (A : 𝕀 → type) (p : (i : 𝕀) → A i) : pathd (symm^1 _ A) (p 1
   | i=1 → refl
   ]
 
+-- transporting backwards is transporting forwards along inverted path (up to composition)
+def coe/symm/d (A : type) (P : A → type) (x y : A) (p : path A x y) (py : P y) :  
```

Comment:
nicely done :sunglasses:

Reply:
yeah I spent some time figuring this one out 😓